quickstart/examples/aggregation/README.md

```diff
@@ -0,0 +1,45 @@
+## Aggregation - Tumble window
+
+- Use this for aggregating data points using a TUMBLE window function (data aggregated for all points in the window at the end of each cycle).
+- Use the windowing function for Tumble window using the keyword TUMBLE_END(datetime, INTERVAL 'duration' unit) & TUMBLE (datetime, INTERVAL 'duration' unit ) in SELECT & GROUP BY section respectively (duration is a number & unit can be SECOND/MINUTE/HOUR).
+
+### Example query
+
+Here booking denotes the booking events stream with sample booking schema.
+
+```SQL
+SELECT
+  count(1) as booking_count,
+  TUMBLE_END(rowtime, INTERVAL '60' SECOND) AS window_timestamp
+from
+  `data_stream_0`
+GROUP BY
+  TUMBLE (rowtime, INTERVAL '60' SECOND)
+```
+
+## Docker Compose Setup
+
+### Prerequisites
+
+1. **You must have docker installed**
+
+Following are the steps for setting up dagger in docker compose -
+1. Clone Dagger repository into your local
+
+   ```shell
+   git clone https://github.com/odpf/dagger.git
+   ```
+2. cd into the aggregation directory:
+   ```shell
+   cd dagger/quickstart/examples/aggregation 
+   ```
+3. fire this command to spin up the docker compose:
+   ```shell
+   docker compose up 
+   ```
+This will spin up docker containers for the kafka, zookeeper, stencil, kafka-producer and the dagger.
+4. fire this command to gracefully close the docker compose:
+   ```shell
+   docker compose down 
+   ```
+   
```

Comment:
Examples or Tutorials are used to demonstrate a particular feature/concept to the user, without divulging too much concepts. Here, the flow can be improved on what the user will be doing or what they will “see” when they run this example, instead of advising on how to use a tumble window, when to use TUMBLE_START, etc. The goal should be to help the user run something quickly and see the results. They can always refer the docs for reference if they don't know the concepts.
@ravisuhag had shared this blog article with me earlier and it is an excellent guide on how to write tutorials: https://documentation.divio.com/tutorials/#

I can recommend this format. You can create these 3 sections maybe:


### About this example

1. Explain what the user will be doing and what they will achieve at the end of this example. Use “we” in your statements so that the person feels inclusive as you guide them through the steps.For example, refer [this line](https://docs.divio.com/en/latest/introduction/django/django-03-setup-project-locally/#:~:text=In%20this%20section%20we%20will%20build%20the%20new%20application%20you%E2%80%99ve%20created%20in%20the%20local%20development%20environment%3B%20that%20is%2C%20we%20will%20set%20it%20up%20on%20your%20own%20computer.)

### Before Trying This Example

1. Mention the setup users need to have on their local: tools, software and versions. For example format, refer this(https://kubernetes.io/docs/tutorials/stateless-application/guestbook/#before-you-begin)

### Steps

1. Mention the steps to be followed here. For each, unless it is very trivial, try to mention what will happen when the command is executed and what output the user will be seeing on the screen. Something like this( https://docs.divio.com/en/latest/introduction/django/django-04-deploy/#:~:text=On%20the%20application%20Dashboard%2C%20you%20will%20see%20that%20your%20new%20commit%20is%20listed%2C%20and%20that%20it%20now%20reports%203%20Undeployed%20commits.%20You%20can%20deploy%20the%20Test%20environment%20using%20the%20Control%20Panel%2C%20or%20by%20running%3A)
2. At the end, mention what is the result and congratulate the user for following through. This makes it encouraging :slightly_smiling_face:  Something like [this](https://docs.divio.com/en/latest/introduction/django/django-04-deploy/#:~:text=Your%20Django%20application%20is%20now%20running%20in%20the%20cloud%20as%20well%20as%20locally%2C%20but%20it%20can%E2%80%99t%20do%20anything%20useful%20until%20it%20is%20attached%20to%20basic%20services%2C%20such%20as%20the%20database%20and%20media%20storage)

Reply:
@sumitaich1998 Apologies for being so verbose on this, but I think if we make the examples smoother, we will have the ability to attract a lot of new users quite quickly. Let me know if you want to pair on this.

Reply:
done